webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtEc2.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtEc2 = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Availability zone')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.availability_zone}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Public IP address')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.public_ip_address}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('DNS name')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.dns_name}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Private IP address')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.private_ip_address}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Private DNS name')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.private_dns_name}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Kernel id')}</DescriptionListTerm>
```

Comment:
One small nit: you use `id` in some translation strings, but I we mostly use ID. Just commenting once here, but it's a repeated thing:
```suggestion
      <DescriptionListTerm>{__('Kernel ID')}</DescriptionListTerm>
```